components/InviteAdminModal/InviteAdminModal.tsx

```diff
@@ -0,0 +1,223 @@
+              <TextField
+                name="email1"
+                value={state.email1}
+                onChange={handleChange}
+                fullWidth
+                autoFocus
+                margin="dense"
+                id="name"
+                placeholder="thisisanemail@email.com"
+                type="text"
+                variant="standard"
+                className={styles['font']}
+              />
+            </div>
+          </div>
+          <div className={styles['row']}>
+            <div className={styles['col']}>
+              <TextField
+                name="name2"
+                value={state.name2}
+                onChange={handleChange}
+                fullWidth
+                autoFocus
+                margin="dense"
+                id="name"
+                placeholder="Firstname Lastname"
+                type="text"
+                variant="standard"
+                className={styles['font']}
+              />
+            </div>
+            <div className={styles['col']}>
+              <TextField
+                name="email2"
+                value={state.email2}
+                onChange={handleChange}
+                fullWidth
+                autoFocus
+                margin="dense"
+                id="name"
+                placeholder="thisisanemail@email.com"
+                type="text"
+                variant="standard"
+                className={styles['font']}
+              />
+            </div>
+          </div>
+          <div className={styles['row']}>
+            <div className={styles['col']}>
+              <TextField
+                name="name3"
+                value={state.name3}
+                onChange={handleChange}
+                fullWidth
+                autoFocus
+                margin="dense"
+                id="name"
+                placeholder="Firstname Lastname"
+                type="text"
+                variant="standard"
+                className={styles['font']}
+              />
+            </div>
+            <div className={styles['col']}>
+              <TextField
+                name="email3"
+                value={state.email3}
+                onChange={handleChange}
+                fullWidth
+                autoFocus
+                margin="dense"
+                id="name"
+                placeholder="thisisanemail@email.com"
+                type="text"
+                variant="standard"
+                className={styles['font']}
+              />
+            </div>
+          </div>
+
+        </div>
+        <div className={styles['actions']}>
+          <DialogActions>
+            <ThemeProvider theme={theme}>
+              <Button style={{ textTransform: 'none' }} className={styles['font']} variant="outlined" onClick={handleClose}>Cancel</Button>
+              <Button style={{ textTransform: 'none' }} className={styles['font']} variant="contained" onClick={handleClose}>Send Invites</Button>
```

Comment:
I think it is possible for you to specify a `text-transform` within the `font` CSS rule, so you don't need to pass in anything for the `style` attribute in the Button component.